AElf.Sdk.CSharp2.Tests/ContractTest.cs

```diff
@@ -3,17 +3,18 @@
 using System.Linq;
 using AElf.Common;
 using AElf.Kernel;
+using AElf.Kernel.Blockchain.Application;
 using AElf.Kernel.SmartContract;
 using Xunit;
 using Shouldly;
+using Moq;
 using AElf.Sdk.CSharp;
 using AElf.Sdk.CSharp2.Tests.TestContract;
 
 namespace AElf.Sdk.CSharp2.Tests
 {
     public class ContractTest
     {
-        /*
         private List<Address> AddressList { get; } = new[] {"a", "b", "c", "d"}.Select(Address.FromString).ToList();
         private IStateManager StateManager { get; } = new MockStateManager();
```

Comment:
IStateProvider you should get service,
GetService<IStateProvider> 

do not need IStateManager

Reply:
@loning Have update code with using GetService instead of IStateProvider and IStateManager.